editor/create_dialog.cpp

```diff
@@ -38,7 +38,7 @@
 #include "editor_settings.h"
 #include "scene/gui/box_container.h"
 
-void CreateDialog::popup_create(bool p_dont_clear, bool p_replace_mode) {
+void CreateDialog::popup_create(bool p_dont_clear, bool p_replace_mode, StringName *class_to_replace) {
```

Comment:
The name of the `class_to_replace` parameters doesn't match the signature in the header.
Also by convention we prefix such arguments changed by the method by `r_`.